Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -1,31 +1,17 @@
-[CmdletBinding(DefaultParameterSetName = 'None')]
-param(
-    [string][Parameter(Mandatory=$true)][ValidateNotNullOrEmpty()] $connectedServiceName,
-    [string][Parameter(Mandatory=$true)][ValidateNotNullOrEmpty()] $projectKey,
-    [string][Parameter(Mandatory=$true)][ValidateNotNullOrEmpty()] $projectName,
-    [string][Parameter(Mandatory=$true)][ValidateNotNullOrEmpty()] $projectVersion,	
-    [string]$dbUrl,
-    [string]$dbUsername,
-    [string]$dbPassword,
-    [string]$cmdLineArgs,
-    [string]$configFile,
-    [string]$includeFullReport,
-    [string]$breakBuild
-)
-
-Write-Verbose "Starting SonarQube Pre-Build Setup Step"
-
-Write-Verbose "connectedServiceName = $connectedServiceName"
```

Comment:
The new Get-VstsInput will output the value to the log so I'm removing the extra logging